core/pageObject/dashbord.ts

```diff
@@ -0,0 +1,61 @@
+import { Locator, Page } from '@playwright/test';
+const logger = require('../logger');
+
+export class Dashboards {
+    readonly page: Page;
+    readonly successfullySignInMsg: Locator;
+    readonly addNewDashboardButton: Locator;
+    readonly newDashboardNameFromPopup: Locator;
+    readonly description: Locator;
+    readonly addDashboardButtonFromPopup: Locator;
+    readonly dashboardAddedSuccessfully: Locator;
+    readonly editDashboardButton: Locator;
+    readonly deleteDashboardButton: Locator;
+    readonly dashboardDeletedSuccessfully: Locator;
+    readonly fullScreenDashboardButton: Locator;
+    readonly addNewWidget: Locator;
+    readonly dashboardNameFromMenu: Locator;
+    readonly confirmDeleteButton: Locator;
+    readonly updateButtonFromEditPopup: Locator;
+
+    constructor(page) {
+        this.page = page;
```

Comment:
You might add an abstract class which all your Page classes would extend. There you might put any common functionality, f.e. this.page = page; setting in constructor. Now you do it in every Page class separately, but if you make them extend some Common class, you might put it there